customer/views.py

```diff
@@ -43,3 +45,16 @@ def post(self, request, *args, **kwargs):
                 return Response(serializer.data, status=status.HTTP_201_CREATED)
         else:
             return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+class CreateOrderView(generics.CreateAPIView):
```

Comment:
shouldn't this be in the orders app 
the creation and updating of orders 
while only the list order of a specific customer should be here

Reply:
why tho

Reply:
do this change and I will merge the branch no more changes needed